### PR TITLE
remove polyfill workarounds

### PR DESCRIPTION
**Pull request by tomalec**
For V0 and V1 quirks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,13 +127,9 @@ Browser support relies mainly on polyfills support/spec compliance.
...
 |:------:|:----:|:----:|:--------:|:--------:|
-|  ✓     | ✓** | ✓*   | ✓*       | ✓*       |
+|  ✓     | ✓**  | ✓    | ✓        | ✓        |
 
-\*
-- **V0**: There is a workaround for [polyfill issue](https://github.com/webcomponents/webcomponentsjs/issues/470), to execute scripts and apply styles define inside the template in imported HTML. Also, some hacks are made to preserve correct position in DOM of scripts executed by polyfill, so `previousSibling` and Polymer's `dom-bind`/`dom-repeat` can be used as in native Web Components, see [more sample use cases](https://github.com/Juicy/imported-template/tree/master/test/use-cases)
-- **V1**: There is also a workaround for [polyfill issue wcjs#872](https://github.com/webcomponents/webcomponentsjs/issues/872)
-
-\*\* **V1**: In IE11 workaround for per-template scripts [polyfill issue wcjs#872](https://github.com/webcomponents/webcomponentsjs/issues/872) does not work
+\*\* **V1**: IE11 does not support `currentScript` so bunch of test for useful cases fails
```
Comment (alshakero) [inline README.md]: "IE11 does not support `currentScript`, so a bunch of tests for useful cases fail."

**Comment by tomalec** [comment]
Lucky you, on my machine local tests for KS were failing ;)

**Comment by warpech** [comment]
It is not ok if tests in KitchenSink fail. In such case, please re-run the tests. If the problem prevails we need to fix it in KitchenSink.

**Comment by tomalec** [comment]
Right now all test are failing in nightly, I bet it's due to web driver incompatibilities, and one fails in Chrome due to Polish locale:
```
3) Failed : KitchenSink.Tests.Test.DatepickerPageTest(Chrome).DatepickerPage_SelectDate
  String lengths are both 7. Strings differ at index 0.
  Expected: "January"
  But was:  "styczen"
  -----------^
at KitchenSink.Tests.Test.DatepickerPageTest.DatepickerPage_SelectDate() in D:\repos\apps\KitchenSink\test\KitchenSink.Tests\Test\DatepickerPageTest.cs:line 37

```

I don't think it's worth investigating deeper before the demo

**Pull request merged**